PWGCF/FemtoDream/femtoDreamPairTaskTrackV0.cxx

```diff
@@ -150,7 +150,7 @@ struct femtoDreamPairTaskTrackV0 {
       trackHistoPartTwo.fillQA(part);
     }
     /// Now build the combinations
-    for (auto& [p1, p2] : combinations(groupPartsOne, groupPartsTwo)) {
+    for (auto& [p1, p2] : combinations(CombinationsFullIndexPolicy(groupPartsOne, groupPartsTwo))) {
```

Comment:
It works exactly as @victor-gonzalez says, so it is a matter of what you want to achieve, what pairs you want to get.

Reply:
Yes, exactly.
The upper-policy makes sense and is necessary if the first particle and the second particle are the same species (e.g. proton proton). But this task shall be used for pairing a track with a v0 candidate. And using the upper-policy here means that we need at least two candidates in the `groupPartsTwo` list. Since these are the v0 candidates, we loose most of our statistics because we consider accidently only events with at least two v0s.

Sorry for the confusion. The note was a bit missleading without the context, I realise now

Reply:
In other words - correct me if I am wrong: groupPartsOne and groupPartsTwo are two different (disjoint?) sets of particles, so `p1p2 != p2p1`

Reply:
exactly. for all the usecases of the here modified task. By desing so to say